mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -74,6 +74,11 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 	FIELD field_25095 DIMENSION_TYPE_KEY Lnet/minecraft/class_5321;
 	FIELD field_25096 BIOME_SOURCE Lnet/minecraft/class_2378;
 	FIELD field_25097 CHUNK_GENERATOR Lnet/minecraft/class_2378;
+	FIELD field_25098 registryKey Lnet/minecraft/class_5321;
+		COMMENT The {@code RegistryKey} representing the ID of the actual registry.
```

Comment:
```suggestion
		COMMENT The {@linkplain RegistryKey} representing the ID of the actual registry.
```

Reply:
Would that actually link it?

Reply:
It should yes

Reply:
It would. The text will just render as plain text (way less discoverable), but you can click it. Doesn't prompt people to click as much.